_cite/cite.py

```diff
@@ -130,6 +130,10 @@
     # source id
     _id = get_safe(source, "id", "").strip()
 
+    # if explicitly flagged, remove/ignore entry
+    if get_safe(source, "remove", False) == True:
+        continue
```

Comment:
I'm not certain, but this seems like it might not rely on the earlier lines of the loop iteration. If that's the case, consider moving this `if` check just below the `log` so it's performed right away without further work. If it does rely on the earlier calls somehow, consider documenting this with a comment.